src/js/tracks/text-track-settings.js

```diff
@@ -327,7 +455,20 @@ class TextTrackSettings extends Component {
   }
 
   /**
-   * Restore text track settings
+   * Sets all <select> elements to their default values.
+   *
+   * @method setDefaults
+   */
+  setDefaults() {
+    Obj.each(selectConfigs, config => {
```

Comment:
I think we typically wrap this in another set of parens in video.js. `Obj.each((selectConfigs, config) =>`